config/config.go

```diff
@@ -34,19 +35,19 @@ func LoadConfig(confName string) (*MergedConfig, error) {
 	return nil, err
```

Comment:
@notzippy You are right. But I think we can just remove `var err error` declaration and `if err == nil` check.
```diff
func LoadContext(confName string, confPaths []string) (*Context, error) {
-	var err error
	for _, confPath := range confPaths {
		conf, err := config.ReadDefault(path.Join(confPath, confName))
		if err == nil {
+			// Line 29:
			return &Context{conf, ""}, nil
		}
	}
-	if err == nil {
-		err = errors.New("not found")
-	}
+	It is guaranteed err is not nil here. Otherwise, we would exit at line 29.
	return nil, err
}
```

For, reference here is an initial version of this function: https://github.com/revel/revel/blob/43d05c83b5464e64329f10aba010351a4fe44156/config.go#L16-L22
The change was introduced by https://github.com/revel/revel/commit/86302320e2b9fe4257f352112ab337b3e2276cd0.